### PR TITLE
GnsGroupReader: fix RSSI conversion

### PR DESCRIPTION
**Pull request by mvglasow**
Maximum observed so far is 0x2C (see #37), thus the upper limit is (probably) at least 0x3F

### CODE REVIEW

File: RDSSurveyor/src/eu/jacquet80/rds/input/GnsGroupReader.java [hunk 1]
```diff
@@ -593,9 +593,9 @@ public void run() {
 								}
 								/*
 								 * TODO not sure if conversion is correct
-								 * (maximum is 0xFF in theory, 0x1F is maximum observed so far)
+								 * (maximum is 0xFF in theory, presumably 0x3F or more, up to 0x2C observed so far)
 								 */
-								gnsData.rssi = intData[6] * 0x842;
+								gnsData.rssi = intData[6] * 0x410;
```
Comment (ChristopheJacquet) [inline RDSSurveyor/src/eu/jacquet80/rds/input/GnsGroupReader.java:598]: I suggest writing this as 0xFFFF / 0x3F. It's clearer and the compiler will fold constants.

**Review by ChristopheJacquet — CHANGES_REQUESTED** [review]
See suggestion below.

**Comment by mvglasow** [comment]
I’ve rewritten the line to make the conversion clearer.

**Comment by ChristopheJacquet** [comment]
Thanks!

**Pull request merged**